src/sage/coding/code_constructions.py

```diff
@@ -732,8 +732,7 @@ def ToricCode(P,F):
          sage: C.minimum_distance()
          24
          sage: C.minimum_distance(algorithm="guava")  # optional - gap_packages (Guava package)
-         ...
-         24
+         ...24
          sage: C = codes.ToricCode([[-2,-2],[-1,-2],[-1,-1],[-1,0],
```

Comment:
Why `...`?

I get just 
```
sage:  C.minimum_distance(algorithm="guava")  
24
```
after installing guava.

Reply:
I would guess it accounts for sporadic output from GAP